crates/oxc_language_server/src/capabilities.rs

```diff
@@ -60,7 +79,8 @@ impl From<Capabilities> for ServerCapabilities {
 mod test {
```

Comment:
I'm not sure the tests are being run. I tried adding `assert!(false)` and didn't get any test to fail... maybe it needs to be enabled in the LS Cargo.toml?

Reply:
Good catch, I didn't noticed it. Feel free to create a bug issue

Reply:
Good catch! I did not notice it. Feel free to create a bug issue

Reply:
I enabled it in this PR since this doesn't seem significant enough to create a separate one.